cfme/tests/configure/test_access_control.py

```diff
@@ -262,9 +262,12 @@ def test_description_required_error_validation():
 
 
 @pytest.mark.tier(3)
+@pytest.mark.meta(blockers=[1351897])
 def test_delete_default_group():
-    flash_msg = \
-        'EVM Group "{}": Error during \'destroy\': A read only group cannot be deleted.'
+    flash_msg = version.pick({'5.6': ("EVM Group \"{}\": Error during delete: A read only " +
```

Comment:
Done for few test cases, but some of the strings are quite long so they do need  the string split.